workspace-manager-clienttests/README.md

```diff
@@ -61,6 +70,35 @@ export TEST_RUNNER_SERVER_SPECIFICATION_FILE="workspace-dev.json"
 Each service account JSON files in the resources/serviceaccounts directory of this project specifies a default file
```

Comment:
*file

Reply:
Corrected plural `files` to singular `file`.